pkg/tfgen/generate.go

```diff
@@ -79,11 +81,12 @@ const (
 	Python Language = "python"
 	CSharp Language = "dotnet"
 	Schema Language = "schema"
+	PCL    Language = "pulumi"
```

Comment:
Why not `pcl`?

Reply:
To be consistent with https://github.com/pulumi/pulumi-terraform-bridge/blob/82448a9d29c68e2831c64f5525ca00e5e00c7a29/pkg/tf2pulumi/convert/convert.go#L40